src/w3iProxy/pushProviders/internalPushProvider.ts

```diff
@@ -118,6 +118,24 @@ export default class InternalPushProvider implements W3iPushProvider {
       throw new Error(this.formatClientRelatedError('subscribe'))
     }
     console.log('InternalPushProvider > PushClient.subscribe > params', params)
+    const currentSubs = Object.values(
+      this.pushClient.getActiveSubscriptions({ account: params.account })
+    )
+
+    /*
+     * We do nothing with this data so it's okay if we return garbage, for now.
```

Comment:
Why do we need this if the data isn't used? This seems to also be creating a mixed return type here which is either `boolean` or this object on the early return?

Again feels like a hack. I see "Move this functionality into notify client." below, but what do we have to add/what does the issue for this look like?

Reply:
It always returns 
```ts
{
  id: string,
  subscribeAuth: string
}
```

I made it return the empty object. I've kept this API to remain compliant with notify client's API. 

I'm still unsure how we should handle it in notify client. Like the comment says, we can't be confident we have the request id, and we definitely can't recreate the original subscription auth. 

Maybe we can make it throw if you try to subscribe to existing sub and handle this in a try catch? We'd still need to return something though.

The other option is having w3i not comply with notify client's API, and return a boolean. I'm open to ideas